infra/aws/terraform/registry-k8s-io-prod/providers.tf

```
@@ -0,0 +1,90 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+terraform {
+  backend "s3" {
+    bucket = "k8s-infra-registry-k8s-io-tf-state"
+    key    = "prod"
+    region = "us-east-2"
+  }
+
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "5.19.0"
+    }
+  }
+}
+
+# Provider for AWS non-region-specific operations
+provider "aws" {
+  region = "us-east-2"
+}
+
+provider "aws" {
+  alias  = "registry-k8s-io-prod"
+  region = "us-east-2"
+
+  assume_role {
+    role_arn = "arn:aws:iam::${local.registry-k8s-io-prod-account-id}:role/OrganizationAccountAccessRole"
```

these all look like good candidates to define a local and then use that local here and throughout below:
```suggestion
    role_arn = local.assume_role_arn
```